lib/daru/date_time/index.rb

```diff
@@ -366,6 +366,24 @@ def [] *key
       end
     end
 
+    def pos *args
+      # to filled
+      out = self[*args]
+      return out if out.is_a? Numeric
+      out.map { |date| self[date] }
+    end
+
+    def subset *args
+      self[*args]
+    end
+
+    def respond? *args
```

Comment:
`respond?` returns true if all arguments is a either a valid index or a valid position.